stack/base/before-notebook.d/40_prepare-aiida.sh

```diff
@@ -48,7 +45,7 @@ if [[ ${NEED_SETUP_PROFILE} == true ]]; then
       exit 1
     fi
 
-    verdi computer show ${computer_name} || verdi computer setup \
```

Comment:
Again, since we have just created a profile, by definition there can't be any computer yet!